erpnext/selling/report/customer_wise_item_price/customer_wise_item_price.py

```diff
@@ -62,10 +62,10 @@ def fetch_item_prices(
 	or_conditions = []
 	if items:
 		and_conditions.append(ip.item_code.isin([x.item_code for x in items]))
-		and_conditions.append(ip.selling is True)
+		and_conditions.append(ip.selling == True)  # noqa: E712
 
-		or_conditions.append(ip.customer is None)
-		or_conditions.append(ip.price_list is None)
+		or_conditions.append(ip.customer == None)  # noqa: E711
+		or_conditions.append(ip.price_list == None)  # noqa: E711
```

Comment:
this a bug. 

`ip.customer == None` converts to `customer == null` which is incorrect. It should've been `ip.customer.isnull()` which converts to `customer is null`